### PR TITLE
HR list of candidates table

### PR DESCRIPTION
**Pull request by islambek01**

### CODE REVIEW

File: src/components/containers/hr/ListOfCandidatesContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        minWidth: 700,
+    },
+    row: {
+        '&:nth-of-type(odd)': {
+            backgroundColor: theme.palette.background.default,
+        },
+    },
+});
+
+let id = 0;
```
Comment (davran-siv) [inline src/components/containers/hr/ListOfCandidatesContainer.jsx:37]: why is the variable created outside the function?

File: src/scenes/head/CreatePosition.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const CreatePosition = () => {
         <div>
             <Header title="Создать Позицию"/>
           
-           {/* <CreatePositionContainer/> */}
+           {/*{ <CreatePositionContainer/> }*/}
```
Comment (davran-siv) [inline src/scenes/head/CreatePosition.js:9]: What is this?

File: src/components/containers/hr/ListOfCandidatesContainer.jsx (hunk 1 above)
Comment (islambek01) [reply src/components/containers/hr/ListOfCandidatesContainer.jsx:37]: If I will take it inside of function id will be always 0(zero).
I can use another way to count it...

File: src/scenes/head/CreatePosition.js (hunk 2 above)
Comment (islambek01) [reply src/scenes/head/CreatePosition.js:9]: Create position not finished yet.

**Pull request merged**